opm/simulators/utils/readDeck.cpp

```diff
@@ -405,16 +419,27 @@ Opm::setupLogging(const int          mpi_rank_,
     }
 
     if (output > FileOutputMode::OUTPUT_NONE) {
-        std::shared_ptr<Opm::EclipsePRTLog> prtLog = std::make_shared<Opm::EclipsePRTLog>(logFileStream.str(), Opm::Log::NoDebugMessageTypes, false, output_cout_);
-        Opm::OpmLog::addBackend("ECLIPSEPRTLOG", prtLog);
-        prtLog->setMessageLimiter(std::make_shared<Opm::MessageLimiter>());
-        prtLog->setMessageFormatter(std::make_shared<Opm::SimpleMessageFormatter>(false));
+        if (mpi_rank_ == 0) {
+            std::shared_ptr<Opm::EclipsePRTLog> prtLog = std::make_shared<Opm::EclipsePRTLog>(logFileStream.str(), Opm::Log::NoDebugMessageTypes, false, output_cout_);
+            Opm::OpmLog::addBackend("ECLIPSEPRTLOG", prtLog);
+            prtLog->setMessageLimiter(std::make_shared<Opm::MessageLimiter>());
+            prtLog->setMessageFormatter(std::make_shared<Opm::SimpleMessageFormatter>(false));
+        }
+        else {
+            auto prtLog = std::make_shared<NullLog>(Opm::Log::NoDebugMessageTypes);
+            Opm::OpmLog::addBackend("ECLIPSEPRTLOG", std::move(prtLog));
```

Comment:
Would have used `move` here as I would assume it only saves an increment. Would rather skip the variable if there are performance concerns.

Reply:
I appreciate the review, but just to be clear: This particular code will never be merged into the master branch.  It is only intended as a proof of concept.